eoscostarica/eos-rate
# feat(webapp): Add Korean Translation

### GH Issue

resolves #460 

### Steps to test

1. open webapp 
1. select `kr` in language selector
1. see translated text

### Checklist
